setup.py

```diff
@@ -30,16 +30,11 @@ def test_suite():
 
 
 major_version, minor_version = sys.version_info[:2]
-if not ((major_version == 2 and minor_version == 7)
-        or (major_version == 3 and minor_version >= 5)):
-    sys.stderr.write("Sorry, only Python 2.7, 3.5, 3.6 and 3.7 are supported "
-                     "at this time.\n")
+if not (major_version == 3 and 5 <= minor_version <= 8):
```

Comment:
In its current state this might cause pain for Python 3.5 users because it will not exit out properly for them. The second comparison should become `6 <= minor_version <= 8` to resolve this. @auvipy thank you for keeping an eye on this repo but the community is much smaller than, say, Celery, so please be less aggressive about merging newly opened PRs here in the future. I ask that we count time-to-merge in days, not hours, because I would like us to catch these kinds of bugs before merging and am interested in thoroughly reviewing every incoming PR [myself](https://jazzband.co/projects/geojson). I will submit a fix for this one if I have time and encourage you to do the same.

Reply:
Thanks for the input. Should we roll back the whole commit 2a87bbd then (including rolling back to Ubuntu 16.04 LTS), since Python 3.5 isn't supported anymore, according to [Travis documentation](https://docs.travis-ci.com/user/reference/bionic/#python-support)?

Reply:
Ah now I see what I did wrong or what you meant, sorry. Should I file a new PR, or will you take care of it @rayrrr?

Reply:
OK

Reply:
I'm sorry. I will be more conservative from now on.